src/ruby/bin/interop/interop_client.rb

```diff
@@ -29,6 +29,12 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+# #######################################################################
+# DEPRECATED: The behaviour in this file has been moved to pb/test/client.rb
+#
+# This file remains to support existing tools and scripts that use it.
```

Comment:
That's going to be done in a follow-up PR.
Mostly related to interop tests; for stability its better to get this change done, update all the references in other places, then remove this file.